bcda/auth/alpha_test.go

```diff
@@ -112,10 +112,17 @@ func (s *AlphaAuthPluginTestSuite) TestDeleteClient() {
 }
 
 func (s *AlphaAuthPluginTestSuite) TestGenerateClientCredentials() {
-	r, err := s.p.GenerateClientCredentials("", 0)
-	assert.Empty(s.T(), r)
+        validClientID := "DBBD1CE1-AE24-435C-807D-ED45953077D3"
+        c, err := s.p.GenerateClientCredentials(validClientID, 0)
```

Comment:
We've started caring more about side effects in our tests.  If we now care that this changes the credentials, we could add code to put things back the way they were.

Reply:
Agree, but this is not trivial since we're dealing with secrets in the DB.